core/src/main/scala/org/apache/spark/executor/CoarseGrainedExecutorBackend.scala

```diff
@@ -283,7 +283,11 @@ private[spark] object CoarseGrainedExecutorBackend extends Logging {
         executorConf,
         new SecurityManager(executorConf),
         clientMode = true)
-      val driver = fetcher.setupEndpointRefByURI(arguments.driverUrl)
+
+      val driver = retry(3) {
```

Comment:
shouldnt this be configurable?

Reply:
Hi @skonto

This class has a Main method and reads the values from the argv. In my opinon is an overkill to add it to define the numbers of the retries. Let me know if what you want is to me add arguments to the main method or you prefer me to leave it as it is.

Reply:
My thought was that someone may want to increase this value because it depends on the env.

Reply:
Yes, I understand, my question still remains. This is part of a Main methof. Do you want me to add it to the args read or not...?

Reply:
I personally would leave this. Not worth and arg or config yet.

Reply:
@srowen  I agree. @skonto  you ok? or how do i move forward, not sure what is the "consensus" for this type of cases.